docs/6.x/pack.md

```diff
@@ -912,3 +912,32 @@ When building a Cluster Image, `tele build` will discover `custom-planet:1.0.0`
 Docker image and vendor it along with other dependencies. During the Cluster
 installation all nodes with the role `worker` will use the custom "planet" Docker
 image instead of the default one.
+
+## Support For Intermediate Upgrades
+
+Kubernetes has a strict version skew [support policy](https://kubernetes.io/docs/setup/release/version-skew-policy/#supported-version-skew) which makes
+cluster upgrades with versions of Kubernetes being more than a couple of minor versions apart complicated if not impossible. This can become a big issue
+if clusters are not kept constantly up-to-date.
+
+Before, the way to upgrade such a cluster had been upgrading by using cluster images with compatible Kubernetes versions in lock step
+until the target image has been applied. This is a time-consuming and not automation-friendly process.
+
+Starting with version `5.5.20`, installers built with `tele` can embed all intermediate versions of Kubernetes necessary to automatically upgrade older
+clusters.
+
+In order to enable intermediate upgrade support, `tele build` should be supplied with a list of Gravity runtime versions to use as intermediate hops.
+For example, to upgrade a cluster based on Gravity `5.0.35` (which is based on Kubernetes `1.9.13`) to a cluster based on Gravity `5.5.20` (which, in turn,
+is based on Kubernetes `1.13.10`), the following flag adds an intermediate step based on Gravity `5.2.15` (which is based on Kubernetes `1.11.9`):
+
+```bash
+$ tele build ... --upgrade-via=5.2.15
+```
+
+!!! note "Intermediate Upgrade Support":
+    Only LTS versions of Gravity can be used as intermediate upgrade hops.
```

Comment:
... "Check [Releases](link) page to see which LTS versions are available".

Also, can the "target" hop (i.e. final upgrade version) be non-LTS, or does it have to be LTS as well? Worth mentioning here I think.